pkg/cmd/roles/list/list.go

```diff
@@ -52,15 +57,15 @@ func NewCmdRoleList() *cobra.Command {
 	return cmd
 }
 
-func roles(numberOfResults string, name string, output bool) {
+func roles(numberOfResults string, name string, output bool, logger zerolog.Logger, ) {
 	var roleInfo role
 	client := util2.GetHttpClient()
 	authToken, apiEndpoint := login.ReadCredentials()
 
 	request, err := http.NewRequest("GET", apiEndpoint+"v1/roles", nil)
 	request.Header.Add("Authorization", authToken)
 	request.Header.Add("Content-Type", "application/json")
-	util2.LogError(err)
+	logging.LogErrorWithMessage("Creating authorization header failed, please review the credentials supplied in sumocli login.", err, logger)
```

Comment:
This is the part I'm not particularly fond of. 

Ideally IMO `logging.LogErrorWithMessage` wouldn't need to take `logger`. The method _could_ lean on the global `log.Logger` but that feels prone to error, so I chose to pass it down instead. 

Potentially we could extend `zerolog.Logger` somehow, but I couldn't find the right way to do that.